cmd/kubeadm/app/util/version.go

```diff
@@ -53,7 +53,8 @@ var (
 func KubernetesReleaseVersion(version string) (string, error) {
 	if kubeReleaseRegex.MatchString(version) {
 		return version, nil
-	} else if kubeReleaseLabelRegex.MatchString(version) {
+	}
+	if kubeReleaseLabelRegex.MatchString(version) {
```

Comment:
I am pretty sure this changes the logic here... We only wanted to reach this `if` if the previous one was not true. In the new code you have, it will always check it even if the first `if` succeeds.